pkg/operator/ceph/object/objectstore_test.go

```diff
@@ -814,3 +814,187 @@ func TestGetRealmKeyArgs(t *testing.T) {
 		assert.Equal(t, "", secret)
 	})
 }
+
+func TestUpdateZoneEndpointList(t *testing.T) {
```

Comment:
Same goes for this test. Is there a reason it's in the `objectstore_test` and not in the zone CRs unit tests?

Reply:
This test calls `ShouldUpdateZoneEndpointList()` which could stay in the objectstore code. @thotz if you move that over to the zone I would think this test should be moved to.